client/components/Entry/EntryComponent.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <article>
+        <tr class="entryLog">
+            <td>{{entry.type}}</td>
+            <td>{{entry.detail}}</td>
+            <td>{{entry.condition}}</td>
+            <td>{{entry.scale}}</td>
+            <td>{{entry.date}}</td>
+            <td v-if="!expanded">
+                <button class="btn-secondary" @click="toggleExpand">
+                    View More
+                </button>
+            </td>
+            <td v-else>
+                <button class="btn-secondary" @click="toggleExpand">
+                    View Less
+                </button>
+            </td>
+            <td>
+                <button class="btn-secondary">
+                    <router-link :to="link">Edit ✏️</router-link>
+                </button>
+            </td>
+            <td>
+                <button class="btn-secondary" @click="deleteEntry">
+                    Delete 🗑️
+                </button>
+            </td>
+        </tr>
+        <div v-if="expanded" class="entryNote">
+            <p>
+                Note: {{entry.notes}}
+            </p>
+        </div>
+    </article>
+</template>
+
+<script>
+export default {
+    name: 'EntryComponent',
```

Comment:
Not sure how to get around this, but basically I got rid of the TypeScript here since `async request(params)` introduced compilation errors? Likely not best practice...